src/UnderDevCrew/Bundle/Model/PlacesModel.php

```diff
@@ -0,0 +1,223 @@
+<?php
+
+namespace UnderDevCrew\Bundle\Model;
+
+
+class PlacesModel
+{
+    /**
+     * @var string
+     */
+    private $id;
+
+    /**
+     * @var string
```

Comment:
Why not :) Moreover it's considered as a float by Google Places itself (according to the format delivered in the response). Nice catch!